tuxemon/npc.py

```diff
@@ -847,5 +847,87 @@ def alter_item_quantity(
 
         return success
 
+    def can_buy_item(self, item_slug: str, qty: int, unit_price: int) -> bool:
+        current_money = self.game_variables.get("money", None)
+        if current_money != None:
```

Comment:
Should be 
`        if current_money is not None: `

https://peps.python.org/pep-0008/#programming-recommendations
"Comparisons to singletons like None should always be done with is or is not, never the equality operators."

Reply:
Same applies for `is True` and `is False`, although you would generally be using

`if foo` rather than `if foo is True`